### ISSUE TITLE
SPC SPC and c-o cannot pop up action menu

### ISSUE THREAD

**Issue by AllenDang** [hlissner/doom-emacs#4058]
**What did you expect to happen?**
SPC SPC and c-o should pop up action menu.

**What actually happened?**
SPC SPC and c-o open the file directly.

**Steps to reproduce:**
1. SPC p p open a project
2. SPC SPC
3. c-o

**System information:**
<details>

```
SYSTEM  type       darwin
        config     x86_64-apple-darwin19.6.0
        shell      /bin/zsh
        uname      Darwin 19.6.0 Darwin Kernel Version 19.6.0: Mon Aug 31 22:12:52 PDT 2020; root:xnu-6153.141.2~1/RELEASE_X86_64 x86_64
        path       (~/.emacs.d/bin/ ~/.yarn/bin ~/.config/yarn/global/node_modules/.bin ~/.nix-profile/bin ~/.emacs.d/bin /usr/local/opt/llvm/bin ~/.cargo/bin ~/SDK/sciter-sdk/bin.osx/ ~/.pub-cache/bin /Library/Frameworks/Python.framework/Versions/3.7/bin /usr/local/sbin ~/SDK/flutter/bin /usr/local/share/dotnet ~/Documents/Gocode/bin /usr/local/apache-ant-1.9.3/bin ~/SDK/android-sdk-macosx/tools ~/SDK/android-sdk-macosx/platform-tools ~/SDK/android-sdk-macosx /usr/local/apache-ant-1.9.3/bin ~/SDK/android-sdk-macosx/tools ~/SDK/android-sdk-macosx/platform-tools ~/SDK/android-sdk-macosx ~/SDK/cocos2d-x-3.7.1/templates ~/SDK/cocos2d-x-3.7.1/tools/cocos2d-console/bin /opt/local/bin /opt/local/sbin ~/.nvm/v0.10.26/bin /usr/local/j2objc-0.9.5 /usr/local/mongodb/bin /usr/local/gradle-2.1/bin /usr/share/java/apache-maven-3.0.5/bin ~/SDK/android-sdk-macosx/platform-tools ~/SDK/android-sdk-macosx/tools ~/Documents/Gocode/bin /usr/local/bin /usr/bin /bin /usr/sbin /sbin /opt/X11/bin /usr/local/git/bin /usr/local/ant/bin /usr/local/go/bin /nix/store/m41ksg1c4p4hh55sdmcy8siy08pdp7z3-emacsGccDarwin/libexec/emacs/28.0.50/x86_64-apple-darwin19.6.0/)
EMACS   dir        ~/.emacs.d/
        version    28.0.50
        build      Oct 09, 2020
        buildopts  --prefix=/nix/store/m41ksg1c4p4hh55sdmcy8siy08pdp7z3-emacsGccDarwin --disable-build-details --with-modules --with-ns --disable-ns-self-contained --with-nativecomp --with-ns CFLAGS=-DMAC_OS_X_VERSION_MAX_ALLOWED=101200
        features   JPEG TIFF PNG RSVG GLIB NOTIFY KQUEUE ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES NATIVE_COMP THREADS JSON PDUMPER
        traits     (server-running envvar-file)
DOOM    dir        ~/.doom.d/
        version    2.0.9
        build      HEAD -> develop 8ba2f15c7 2020-10-08 16:39:45 -0400
        elc-files  0
        modules    (:completion company (ivy +icons +prescient +childframe) :ui doom doom-dashboard hl-todo modeline ophints (popup +all +defaults) treemacs vc-gutter (window-select +numbers) workspaces :editor (evil +everywhere) file-templates (format +onsave) multiple-cursors rotate-text snippets word-wrap :emacs dired electric undo vc :term vterm :checkers syntax :tools (eval +overlay) lookup lsp :os macos :tools magit rgb taskrunner :lang (dart +flutter +lsp) emacs-lisp (go +lsp) json (javascript +lsp) lua markdown org rest sh web yaml :config (default +bindings +smartparens))
        packages   ((lsp-ui :disable t) (literate-calc-mode) (tiny))
        unpin      (n/a)
        elpa       (n/a)
```
</details>

**Comment by drchsl** [comment]
Can confirm. This only happens with `(ivy +childframe)`.

**Comment by hlissner** [comment]
This seems to be an incompatibility with `ivy-childframe`, but haven't had the time to determine if the issue is internal (in Doom) or external (in ivy-childframe).

**Comment by drchsl** [comment]
Not sure if it's of any help, but:
 when i remove the binding for `ivy-dispatching-done`, C-o triggers the hydra (not the menu as before).
 https://github.com/hlissner/doom-emacs/blob/8024f4edf918a1c909ff5edc4f35b8f7f7dbff4e/modules/completion/ivy/config.el#L108

**Comment by hlissner** [comment]
Turns out this was an ivy-childframe issue, which was fixed upstream in tumashu/ivy-posframe@d16fbb0. As of dc50c6b the `:completion ivy` module now pins to a commit of ivy-childframe that includes this fix, so this issue is resolved. Let me know if that isn't the case and I'll reopen it. Thanks for bringing it to my attention!
